tutugodfrey/maintenance-tracker
# feat(approve-request): admin can set the status of request to approve

#### What does this PR do?
create endpoint that allows an admin to approve a request

#### Description of Task to be completed?
define put /requests/requestId/appprove 
create controller to handle the route

#### How should this be manually tested?
use postman to test the endpoint on https://mtrackers.herokuapp.com/api/v1/requests/requestId/approve
ensure isAdmin is set to true
ensure requestId is pass req params



### What are the relevant pivotal tracker stories?
#157824412
